src/__tests__/articleTests/viewArticle.test.js

```diff
@@ -53,4 +53,6 @@ it("viewing article", () => {
   };
   let component = shallow(<ViewArticle {...props} />);
   component;
+  component.instance().componentDidMount();
```

Comment:
Write actual tests. When you're writing tests, you often need to check that values meet certain conditions using expect.